arrows/vtk/applets/color_mesh.cxx

```diff
@@ -480,7 +480,9 @@ class color_mesh::priv : public kwiver::arrows::vtk::mesh_coloration
       }
     }
 
-    kwiver::arrows::pdal::save_point_cloud_las(output_path, lgcs, points, colors);
+    kwiver::arrows::pdal::pointcloud_io pc_io;
```

Comment:
This is perfectly fine for this PR, but the ultimate goal of this refactoring is to dynamically construct the `pc_io` object from a config file that contains the string "pdal" and thereby remove any direct dependency of vtk applets on pdal.  That change can come in a follow-up PR.